14th-team5-iOS/Core/Sources/Bibbi/BBStorages/UserDefaultsWrapper/UserDefaultsWrapper.swift

```diff
@@ -13,12 +13,12 @@ final public class UserDefaultsWrapper {
     
     public static let standard = UserDefaultsWrapper()
     
-    private let userDefaults: UserDefaults!
+    private let userDefaults: UserDefaults
     
     private(set) public var suitName: String
     
     private static let defaultSuitName: String = {
-        "UserDefaultsWrapper"
+        "com.5ing.bibbi"
     }()
```

Comment:
경미님 저 부분을 "com.5ing.bibbi" 로 하드코딩 하지말고 아래와 같이 하는건 어떤가용?
- identifier가 변경되거나 identifier가 수정되오 `Userdefaults` 내부 데이터를 가져오지 못 할경우 사용자에게 데이터를 못보여주는 것 보다 종료시키는게 좋을 것 같아서 
```suggestion
private static let defaultSuitName: String = {
    guard let bundleID = Bundle.main.bundleIdentifier else {
        assertionFailure("Bundle Identifier가 존재하지 않습니다.")
        return "com.5ing.bibbi"
    }
    return bundleID
}()

```

Reply:
조아유